commands/credentials.js

```diff
@@ -51,11 +52,36 @@ function * run (context, heroku) {
         return renderAttachment(attachment, app, isLast)
       })
 
-      return [cred].concat(attLines).join('\n')
+      let rotationLines = []
+      let credentialStore = credentials.filter(a => a.name === cred)[0]
+      if (credentialStore.state === 'rotating') {
+        let formatted = credentialStore.credentials.map(function (credential, idx) {
+          return {
+            'user': credential.user,
+            'state': credential.state,
+            'connections': credential.connections
+          }
+        })
+        let connectionInformationAvailable = formatted.some(function (c) { return c.connections })
+        if (connectionInformationAvailable) {
+          let prefix = '       '
```

Comment:
This seems like a pretty brittle way to handle this--if the ` ├─ as ` formatting changes at all, this will break. On the other hand, I don't have a great suggestion; maybe this is the best we've got for now.